src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -254,15 +354,24 @@ def _run(args: argparse.Namespace, methods: Sequence[str],
     logger.info("Starting python xDS Interop Client.")
     global _global_server  # pylint: disable=global-statement
     method_handles = []
-    for method in methods:
-        method_handles.append(
-            _MethodHandle(method, per_method_metadata.get(method, []),
-                          args.num_channels, args.qps, args.server,
-                          args.rpc_timeout_sec, args.print_response))
+    channel_configs = {}
+    for method in _SUPPORTED_METHODS:
+        if method in methods:
+            qps = args.qps
+        else:
+            qps = 0
```

Comment:
Do we want to raise an exception here?

Reply:
C-core, Ruby and Java put an assertion. It's a nice-to-have thing. Not a big issue, more for preventing bugs.

Reply:
I don't understand why there would be an assertion here. The intent is to create threads for all RPC types from the beginning of the process, but to set the QPS rate for those not yet configured to 0.

This is not an exceptional codepath. In fact, by default, it will be run every time the process starts up.

Reply:
Sorry, I misunderstood the logic here. The unsupported method will be checked by `parse_rpc_arg`.